### DISCUSSION TITLE
proper configuration to get the client remote ip logged by Nextcloud

### DISCUSSION THREAD

**Discussion by Omaha2002** [nextcloud/all-in-one#4338] [Questions] [answered]
I've had a proper configuration with a local installation of Nextcloud and a pfsense firewall forwarding the client-ip to the Nextcloud instance. Handy for bruteforce detection.

Trying to accomplish this in Nextcloud AIO means editting conf files in the docker containers. 

Is it possible to have the proper apache settings installed by Nextcloud AIO? So that adding send-proxy-v2 to HA proxy would be enough to have the real-ip logged in Nextcloud logs.

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/2045?
